cpp/src/io/utilities/parsing_utils.cuh

```diff
@@ -153,6 +153,11 @@ parse_numeric(const char* data, long start, long end, ParseOptions const& opts,
     start++;
   }
 
+  // Skip over the "0x" prefix for hex notation
+  if (base == 16 && start + 2 <= end && data[start] == '0' && data[start + 1] == 'x') {
```

Comment:
I still think it would be worth adding the prefix string to ParseOptions instead of hardcoding it here, but I suppose this is fine for now.

Reply:
Not sure I'd want to have more stuff in ParseOption (the string is tied to the base). My primary concern with the whole parsing code is that it's an order of magnitude slower than it should be.